code/API_definitions/quality-on-demand.yaml

```diff
@@ -872,6 +924,22 @@ components:
         - code
         - message
 
+    RetrieveSessionsInput:
+      description: Parameters to get QoS session information by device
+      type: object
+      required:
+        - device
```

Comment:
device should be optional, not required

Reply:
I haven't read the last [comment](https://github.com/camaraproject/QualityOnDemand/issues/101#issuecomment-2232978314) in #101 before making my comment. Fine for me to keep device in this PR required.